<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalansAddSets</class>
 <widget class="QWidget" name="BalansAddSets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>511</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="mainLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Наименование:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdSave">
          <property name="text">
           <string>Сохранить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmdDrop">
          <property name="text">
           <string>Сбросить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Действия:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="txtName"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="rightMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QPushButton" name="cmdAddFactory">
            <property name="text">
             <string>Добавить группу</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdAddFactory</sender>
   <signal>clicked()</signal>
   <receiver>BalansAddSets</receiver>
   <slot>appendChild()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>778</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdSave</sender>
   <signal>clicked()</signal>
   <receiver>BalansAddSets</receiver>
   <slot>saveChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>773</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdDrop</sender>
   <signal>clicked()</signal>
   <receiver>BalansAddSets</receiver>
   <slot>dropChanges()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>815</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>appendChild()</slot>
  <slot>saveChanges()</slot>
  <slot>dropChanges()</slot>
 </slots>
</ui>
